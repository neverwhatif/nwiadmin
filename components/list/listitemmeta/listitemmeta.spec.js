import React from 'react';
import { shallow } from 'enzyme';

import ListItemMeta from './index';

describe('ListItemMeta', () => {
    const component = shallow(<ListItemMeta data={['Lorem', 'Ipsum']} />);

    it('should render', () => {
        expect(component).toMatchSnapshot();
    });

    it('should render a list', () => {
        expect(component.find('li').length).toBe(2);
    });
});
