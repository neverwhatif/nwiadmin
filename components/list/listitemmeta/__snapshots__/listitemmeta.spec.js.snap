// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItemMeta should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ListItemMeta
    data={
      Array [
        "Lorem",
        "Ipsum",
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <li
          className="item"
        >
          Lorem
        </li>,
        <li
          className="item"
        >
          Ipsum
        </li>,
      ],
      "className": "root",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "host",
        "props": Object {
          "children": "Lorem",
          "className": "item",
        },
        "ref": null,
        "rendered": "Lorem",
        "type": "li",
      },
      Object {
        "instance": null,
        "key": "1",
        "nodeType": "host",
        "props": Object {
          "children": "Ipsum",
          "className": "item",
        },
        "ref": null,
        "rendered": "Ipsum",
        "type": "li",
      },
    ],
    "type": "ul",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <li
            className="item"
          >
            Lorem
          </li>,
          <li
            className="item"
          >
            Ipsum
          </li>,
        ],
        "className": "root",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "host",
          "props": Object {
            "children": "Lorem",
            "className": "item",
          },
          "ref": null,
          "rendered": "Lorem",
          "type": "li",
        },
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "host",
          "props": Object {
            "children": "Ipsum",
            "className": "item",
          },
          "ref": null,
          "rendered": "Ipsum",
          "type": "li",
        },
      ],
      "type": "ul",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
